tree-sitter/src/TreeSitter/Unmarshal/Examples.hs

```diff
@@ -62,20 +63,10 @@ instance SymbolMatching Bin where
   showFailure _ _ = ""
 
 -- | Anonymous leaf node.
-newtype AnonPlus a = AnonPlus { ann :: a }
-  deriving (Generic1, Unmarshal)
-
-instance SymbolMatching AnonPlus where
-  symbolMatch _ _ = False
-  showFailure _ _ = ""
+type AnonPlus = Token "+" 0
 
 -- | Anonymous leaf node.
-newtype AnonTimes a = AnonTimes { ann :: a }
-  deriving (Generic1, Unmarshal)
-
-instance SymbolMatching AnonTimes where
-  symbolMatch _ _ = False
-  showFailure _ _ = ""
+type AnonTimes = Token "*" 1
```

Comment:
We don’t actually have a grammar to go along with these examples, so I’m just pretending that `0` and `1` are valid symbols for them, just like the old `SymbolMatching` instances just returned arbitrary values but couldn’t actually do anything useful.